assets/js/blocks/featured-product/block.js

```diff
@@ -44,7 +44,7 @@ import {
 	getImageSrcFromProduct,
 	getImageIdFromProduct,
 } from '../../utils/products';
-import withProduct from '../../utils/with-product';
+import withProduct from '../../hocs/with-product';
```

Comment:
ya looks good, however in combination with [my branch](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/791) we'll probably want to export this from `index.js` in the hocs folder right?

Reply:
Done in 35918e9ca829d462d9d16643539616eb28d5ab07. :+1: